test/UnitTests/Microsoft.Testing.Extensions.VSTestBridge.UnitTests/ObjectModel/ObjectModelConvertersTests.cs

```diff
@@ -20,23 +20,6 @@ public ObjectModelConvertersTests(ITestExecutionContext testExecutionContext)
     {
     }
 
-    public void ToTestNode_WhenTestCaseHasTestNodeUidProperty_TestNodeUidUsesIt()
```

Comment:
Why have you removed them instead of updating them to show we use the ID?

Reply:
I wonder what this test is asserting. If we change anyway means we expect different value so I thought we're not checking nothing actually. But I can move it back if you want.

Reply:
Let's remove the test.